solutions/silver/usaco-645.mdx

```diff
@@ -6,32 +6,6 @@ title: Splitting the Field
 author: Óscar Garries, Benjamin Qi, Ryan Chou
 ---
 
-[Official Analysis (Java)](http://www.usaco.org/current/data/sol_split_gold_open16.html)
```

Comment:
ayo what

Reply:
Im not sure what is causing the failure I am just removing parts to find what it is.